tests/test_perf_tracking.py

```diff
@@ -2227,26 +2227,6 @@ def test_empty_positions(self):
             self.assertEquals(val, 0)
             self.assertNotIsInstance(val, (bool, np.bool_))
 
-    def test_update_last_sale(self):
```

Comment:
The `update_last_sale` method is going away (already removed on lazy-mainline branch). Also the current test only asserts on the cash adjustment, and doesn't check the last_sale_price. So the trade off there is adding a coverage on something that is going away, versus clearing the way.